### ISSUE TITLE
Unable to view calculated fields or modified date fields from SharePoint List

### ISSUE THREAD

**Issue by gradebg96** [dataiku/dss-plugin-sharepoint-server#8]
Hello, is this plugin pulling data from the Default view in Sharepoint or somewhere else?  I am unable to see system generated fields (like Modified Date) or Calculated fields in Dataiku upon connecting.  Selection fields, text, and number fields work fine.  Would this be a configuration error?

**Comment by alexbourret** [comment]
Hi,
Sorry for the wait...
The [v1.0.3-beta.1](https://github.com/dataiku/dss-plugin-sharepoint-server/releases/tag/v1.0.3-beta.1) should let you access fields such as Creation Date, Modified Date, Created by and Modified by.
The calculated fields should be visible when **Expand lookup fields** is ticked.
![lookup_fields](https://user-images.githubusercontent.com/3763569/123087598-06545280-d425-11eb-8daf-bebc3c05f8f9.png)
Please let us know if you are still experiencing any issue.

**Comment by gradebg96** [comment]
Fantastic, we'll install the beta and see if any issues

**Comment by alexbourret** [comment]
Hi @gradebg96 !
Did this beta fix your calculated field issue in the end ? Did you experience any issue with this version of the plugin ?

**Comment by gradebg96** [comment]
Hello Alex, thank you for coming back. Finally got this setup in prod just recently. So far I'm able to see the calculated fields now and the date metadata fields. I'm not able to see data in any fields that indicate a person, such as "modified by". I'll check again shortly as I'm not sure if this is a configuration issue on our side or not yet

**Comment by gradebg96** [comment]
Hello @alexbourret I've had my team now verify they are unable to see "People".  We can now see calculated fields and dates correctly, but company users aren't showing up.  This means if we create a sharepoint field for an Approver, as an example, that name shows up Blank in DSS.  Similar to the metadata for Created By or Modified By, we can we when it was modified (date), but not who it was modified by (user).

**Comment by alexbourret** [comment]
Hi @gradebg96 ! Can you confirm that you are using the 1.0.3 beta version, and the settings for the dataset are similar to what's on the screenshot ?
![sharepoint_server](https://user-images.githubusercontent.com/3763569/167107841-33a4f483-dd21-4c10-b23d-0994d758f502.png)
(`expand lookup fields` activated, and `Modified by` selected)

**Comment by gradebg96** [comment]
Yes correct. The fix has worked for the dates and calculated fields, but not for people (or lookup values)
